test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -351,7 +351,7 @@ public async Task GivenASearchParameterWithMultipleBaseResourceTypes_WhenTargeti
             }
         }
 
-        [SkippableFact]
+        [Fact(Skip = "Re-enable this test when https://microsofthealth.visualstudio.com/Health/_boards/board/t/Olympus/Stories/?workitem=83531 is fixed")]
```

Comment:
@feordin As discussed, I have disabled tests in this PR and we would need to reenable when the correspoinding workitem is fixed. Thanks!